### ISSUE TITLE
Unloader upgrade: plastainum/phase unloader

### ISSUE THREAD

**Issue by AlvarGD** [Anuken/Mindustry-Suggestions#2013]
DAMMIT I ACCIDENTALLY SENT THIS BEFORE FINISHING THE TITLE ITS SUPPOSED TO SAY UNLOADER UPGRADE

**Describe the content or mechanics you are proposing.**
An unloader upgrade, called Phase/Plastanium unloader using phase, thorium, plastanium and silicon that requires 120 power to function. This unloader is just fast enough to fill up a whole plastanium belt. It unloads 8 times a second in bulks of 5, kinda like the plastanium belt. This one is also 1x1, and it takes items from the one with the least items to the one with the most.

**Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.**
The current unloaders only go at around 8 items per second, and with how Mindustry is now, that is SLOW. This unloader doesn't render the other one useless as it is expensive to make and requires power, but it still is very important because it could be used, for example, in defenses. This unloader can very quickly distribute all of the ammo from a vault, without forcing you to use a lot of it. It could also be used together with the plastanium belts to fill up large factories that need many resources or fill up the unit makers that use tons of resources.

**Before making this issue, place an `X` in the boxes below to confirm that you have acknowledged them.** *Failure to do so may result in your request being closed automatically.*

1. - [X] I have done a quick search in the list of suggestions to make sure this has not been suggested yet.
2. - [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
3. - [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
4. - [X] I have read `README.md` to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by joshuaptfan** [comment]
1. You can edit issues even after submitting.
2. Use some creativity in your designs instead of asking for everything to be buffed.

**Comment by AlvarGD** [comment]
I cant see how to edit the title. Also its not just a buff. Its got a lot of creativity. Its an unloader but it works like a plastanium conveyor, with the bulk things and it uses power. Its also later in the game and it has this fancy way of determining priority.

**Comment by itcannotbe** [comment]
Why do we need something overpowered again?

**Comment by joshuaptfan** [comment]
On the right of the title, next to the green "New Issue" button, is an "Edit" button for the title.

**Comment by itcannotbe** [comment]
Also you can't fill up a plastanium conveyor from only one loading point. You just can't intentionally, by design.

**Comment by joshuaptfan** [comment]
^ Even item sources in sandbox, which output items at an infinite rate, cannot saturate a plastanium conveyor without a T-merge of 2 inputs.

**Comment by AlvarGD** [comment]
i know but you can put two if them really close togheter from one instead of needing 5 of them and obstructing presious space. Also plastanium conveyors with 40 items per second and mass drivers with OVER A HUNDRED items per second are ok but an unloader with 40 items per second output, very useful for defenses and making big factories that requires endgame materials and 120 power is not ok?

**Comment by itcannotbe** [comment]
Using power to deter using an item is the worst kind of deterrent.

120 power consumption is a joke.

**Comment by itcannotbe** [comment]
> ^ Even item sources in sandbox, which output items at an infinite rate, cannot saturate a plastanium conveyor without a T-merge of 2 inputs.

Not exactly, they (item sources) too are capped

**Comment by AlvarGD** [comment]
so you want moar power consumption? also did you forget the whole endgame items thing?

**Comment by AlvarGD** [comment]
Also it could be like 20 items per second or something instead idk it doesnt have to be those exact numbers

**Comment by AlvarGD** [comment]
> > ^ Even item sources in sandbox, which output items at an infinite rate, cannot saturate a plastanium conveyor without a T-merge of 2 inputs.
> 
> Not exactly, they (item sources) too are capped

by your fps

**Comment by joshuaptfan** [comment]
Plastanium conveyors and mass drivers as special item transport methods. The titanium conveyor throughput is the "standard" throughput — that's the throughput that junctions and bridge conveyors also have.

Just like there are no plastanium junctions, plastanium routers, plastanium bridges, etc, there should not be plastanium unloaders, because it would severely bloat the distribution menu. Plastanium conveyors are not meant to be a straight upgrade from titanium conveyors, but its own special method of transport requiring creative workarounds to use.

**Comment by Kyllingene** [comment]
*mmm* this seems like a doomed suggestion. Honestly, it can't be *that* hard to change your designs enough to fit plastanium conveyors. Either the power consumption will not be nearly enough, or it will be an insane amount for a distribution block. The only reason it works for phase conveyors is because it would be a pain to link power nodes along the entire chain if you tried to use them long-range.

**Comment by AlvarGD** [comment]
ok how about they can only go to plastanium conveyors or things with storage

> Plastanium conveyors and mass drivers as special item transport methods. The titanium conveyor throughput is the "standard" throughput — that's the throughput that junctions and bridge conveyors also have.
> 
> Just like there are no plastanium junctions, plastanium routers, plastanium bridges, etc, there should not be plastanium unloaders, because it would severely bloat the distribution menu. Plastanium conveyors are not meant to be a straight upgrade from titanium conveyors, but its own special method of transport requiring creative workarounds to use.

ok how about they can only go to plastanium conveyors or things with storage

**Comment by itcannotbe** [comment]
Also you intend to, still make, a block that can output 120 items per suck ond, just be a fucking 1x1.

Now that's overpowered on it's own.

**Comment by genNAowl** [comment]
Yeah unloaders are already extremely powerful, a 1x1 block that unloads more shouldn't exist.

**Comment by RedstonerKBC** [comment]
> Why do we need something overpowered again?

Foreshadows and Quads: No reason why. OP things are good. They make the game healthier.

**Comment by itcannotbe** [comment]
Foreshadows are meant to deal with T5, heck it has a nerf when shooting buuldings. So no, might seem overpowered but it it's not really.

**Comment by itcannotbe** [comment]
Also see:

Corre snek
Sorter snek
Sorter Flowgate snek

I'm sorry, are you having trouble finding them?
Well because they're overpowered and those mechanics were removed from the game.

**Comment by kapzduke** [comment]
> 1. You can edit issues, including the title, even after submitting.
> 2. Use some creative problem solving in your designs instead of asking for everything to be buffed.
> 3. Upvoting your own issue makes you look pathetic.

4. MOOOOOODS MOOOOOOOOOODS

**Comment by Kyllingene** [comment]
> Also plastanium conveyors with 40 items per second and mass drivers with OVER A HUNDRED items per second are ok

Because both of them are limited in their use range. Can you make a junction with plast conveyors? Only a large one. Can you use Mass Drivers to saturate a plast line? Only in specific circumstances. Can you use the suggested unloader exactly like the worse one? Yes, you can. The only downside is power consumption, which is minimal. One steam gen could cover 4 plast unloaders, and who only has steam gens? At that stage, you have at least Thorium reactors, so you can easily cover a measly 120 cost per unloader.

TL;DR: Plast conveyors are harder to use, mass drivers are highly specialized, but these unloaders are simply an unnecessary, direct upgrade.
